client/src/utils/track.ts

```diff
@@ -7,6 +7,14 @@ let configured = false;
 if (!configured) {
   Auth.configure(awsexports);
   Analytics.configure(awsexports);
+  // @ts-ignore
+  if (typeof AWSCShortbread !== "undefined") {
+    // @ts-ignore
+    const shortbread = AWSCShortbread({
+      domain: ".amplify.aws",
+    });
+    shortbread.checkForCookieConsent();
```

Comment:
Is shortbread used anywhere else within this file?

Reply:
Think that's it -- Shortbread does a lot automatically

Reply:
can you shorten this to be a one line then? 
```suggestion
    AWSCShortbread({ domain: ".amplify.aws" }).checkForCookieConsent();
```

Reply:
ooh probably.. not sure why I didn't see this before

Reply:
fixed!